great-idea/css/index.css

```diff
@@ -352,13 +352,15 @@ ul {
 	padding-top: 65px;
 }
 
-.button1 {
-	background-color: #4CAF50; /* Green */
-    border: none;
-    color: white;
-    padding: 15px 32px;
-    text-align: center;
-    text-decoration: none;
-    display: inline-block;
-    font-size: 16px;
+.myButton {
```

Comment:
I like how your the ease transition your buttons have. Your services page looks great.